src/__tests__/reporter/eventListeners.spec.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+import { EVENTS } from '@reportportal/client-javascript/lib/constants/events';
+import { setupReporter } from '../mocks/ReporterMock';
+import { Reporter } from '../../reporter';
+
+describe('test listeners', () => {
+  let reporter: Reporter;
+  const startTime = Date.now();
+  const listeners = [EVENTS.SET_LAUNCH_STATUS, EVENTS.SET_STATUS];
+
+  beforeEach(() => {
+    reporter = setupReporter(['launchId', 'suiteIds']);
+    reporter.reportTaskStart(startTime, undefined, undefined);
+  });
+
+  afterAll(() => {
+    process.removeAllListeners();
```

Comment:
It may affect other events on `process`. It will be better to remove listeners inside of reporter after finish launch phase.